docs/guides/hedgehog-engine/unleashed/levels/glvl.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Sonic GLvl setup tutorial for Sonic Unleashed
+---
+
+# How to use Sonic GLvl with Sonic Unleashed
+
+!!! note
+    This guide covers how to get unleashed levels working in order to edit them, this guide does not cover how to use GLvl. its recommended you also read the [Generations GLvl guide.](/guides/hedgehog-engine/blueblur/levels/glvl.md){ target="_blank"}
+
+
+    
+!!! info
+
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+    
+    - [Unleashed Mod Toolbox](/tools/hedgehog-engine/unleashed/files){ target="_blank"}  
+
+    - [Unleashed Object Template](assets/glvl/UnleashedObjectTemplate.zip)
+
+    - Xbox 360 SDK, find this yourself
+
+
+
+
+
+
+
+#### Part 1 (Preparing .ar Files)
+This example will be editing  Windmill Isle Act 1 Day. Open your extracted unleashed game folder and locate your files. In my case the files are #ActD_MykonosAct1.ar.00, #ActD_MykonosAct1.arl, ActD_MykonosAct1.ar.00, ActD_MykonosAct1.ar.01, ActD_MykonosAct1.arl.
+
+Once located you will need to copy those files to the #files folder inside your Mod Toolbox directory. now open #SUModToolbox.bat and press 3 and enter to decompress the files, press 1 or 2 depending on the system and press enter.
+ 
+Now copy the files inside #output/DecompressedFiles to a seperate folder. In order for GLvl to read the model you need to make a folder called "Packed" and inside your unleashed game folder go to "Packed" and copy the level folder to the new "Packed" folder next to your decompressed .ar files.
+
+#### Part 2 (Preparing GLvl)
+Locate your Xbox 360 SDK's bin/win32 folder and copy xbcompress.exe, xbdecompress.exe, xbdm.dll, msvcp71.dll and msvcr71.dll and paste them into your GLvls "bin" directory.
+
+Open your GLvl folder and go to /database/LevelDatabase.xml, you will need to add your entry for the level, in my case it would be:
+
+"Entry name="ActD_MykonosAct1" geometry="ActD_MykonosAct1"    layout_merge=""     slot="WindmillIsle 1"     game="Unleashed""
```

Comment:
Put this inside a code block (not an inline one), and add an example of how the file should look like after adding the game.